### PR TITLE
Remove select and apply make format

### PR DESCRIPTION
**Pull request by snwoods**

### CODE REVIEW

**Comment by psafont** [comment]
Please add polly dependency in the uniext opam metadata, it would be great to have unit tests for unixext as well

**Comment by psafont** [comment]
Please separate the reformatting and the removal of select into different commits

File: lib/xapi-stdext-unix/unixext.ml [hunk 1]
```diff
@@ -419,19 +419,30 @@ let proxy (a : Unix.file_descr) (b : Unix.file_descr) =
...
+      let epoll = Polly.create () in
+      List.iter (fun fd -> Polly.add epoll fd Polly.Events.inp) r ;
+      List.iter (fun fd -> Polly.add epoll fd Polly.Events.out) w ;
+      Fun.protect
+        ~finally:(fun () -> Polly.close epoll)
+        (fun () ->
+          ignore
+          @@ Polly.wait epoll 4 (-1) (fun _ file_desc event ->
+                 (* Do the writing before the reading *)
+                 if event = Polly.Events.out then
```
Comment (lindig) [inline lib/xapi-stdext-unix/unixext.ml:431]: This could be simplified to `if event = Polly.Events.out && a = file_desc then ... ;`

**Review by lindig — APPROVED** [review]

**Comment by edwintorok** [comment]
Moved to https://github.com/xapi-project/xen-api/pull/5402

**Pull request closed without merging**